### PR TITLE
Doc: add CVaaS Regional URLs

### PR DESCRIPTION
**Pull request by noredistribution**
## Change Summary

adding CVaaS regional URLs to documentation

## Component(s) name

`arista.avd.eos_config_deploy_cvp`

### User Checklist

- N/A

### Repository Checklist

- [ x] My code has been rebased from devel before I start
- [x ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x ] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_config_deploy_cvp/docs/avd-to-cvaas.md [hunk 1]
```diff
@@ -72,6 +72,20 @@ ansible_httpapi_validate_certs: True
...
+cluster URLs used in production:
+
+| Region | URL |
+|--------|-----|
+| United States 1a | [www.arista.io](www.arista.io) |
+| United States 1a| [www.cv-prod-us-central1-c.arista.io](www.cv-prod-us-central1-c.arista.io)|
+| Canada | [www.cv-prod-na-northeast1-b.arista.io](www.cv-prod-na-northeast1-b.arista.io)|
+| Europe West 2| [www.cv-prod-euwest-2.arista.io](www.cv-prod-euwest-2.arista.io)|
+| Japan| [www.cv-prod-apnortheast-1.arista.io](www.cv-prod-apnortheast-1.arista.io)|
+| Australia | [www.cv-prod-ausoutheast-1.prod.arista.io](www.cv-prod-ausoutheast-1.prod.arista.io)|
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_config_deploy_cvp/docs/avd-to-cvaas.md]:
```suggestion
| United States 1a | [www.arista.io](https://www.arista.io) |
| United States 1a| [www.cv-prod-us-central1-c.arista.io](https://www.cv-prod-us-central1-c.arista.io)|
| Canada | [www.cv-prod-na-northeast1-b.arista.io](https://www.cv-prod-na-northeast1-b.arista.io)|
| Europe West 2| [www.cv-prod-euwest-2.arista.io](https://www.cv-prod-euwest-2.arista.io)|
| Japan| [www.cv-prod-apnortheast-1.arista.io](https://www.cv-prod-apnortheast-1.arista.io)|
| Australia | [www.cv-prod-ausoutheast-1.prod.arista.io](https://www.cv-prod-ausoutheast-1.prod.arista.io)|
```

**Review by ClausHolbechArista — APPROVED** [review]

File: ansible_collections/arista/avd/roles/eos_config_deploy_cvp/docs/avd-to-cvaas.md [hunk 2]
```diff
@@ -72,6 +72,20 @@ ansible_httpapi_validate_certs: True
...
+| Region | URL |
+|--------|-----|
+| United States 1a | [www.arista.io](https://www.arista.io) |
+| United States 1c| [www.cv-prod-us-central1-c.arista.io](https://www.cv-prod-us-central1-c.arista.io)|
+| Canada | [www.cv-prod-na-northeast1-b.arista.io](https://www.cv-prod-na-northeast1-b.arista.io)|
+| Europe West 2| [www.cv-prod-euwest-2.arista.io](https://www.cv-prod-euwest-2.arista.io)|
+| Japan| [www.cv-prod-apnortheast-1.arista.io](https://www.cv-prod-apnortheast-1.arista.io)|
+| Australia | [www.cv-prod-ausoutheast-1.arista.io](https://www.cv-prod-ausoutheast-1.arista.io)|
+
+> NOTE That URLs without `wwww` are not supported.
```
Comment (carl-baillargeon) [inline ansible_collections/arista/avd/roles/eos_config_deploy_cvp/docs/avd-to-cvaas.md]:
```suggestion
> NOTE That URLs without `www` are not supported.
```
I think I would put this as !!! warning for better visibility.

Comment (noredistribution) [reply ansible_collections/arista/avd/roles/eos_config_deploy_cvp/docs/avd-to-cvaas.md]: Done!

**Review by carl-baillargeon — APPROVED** [review]
LGTM

**Pull request merged**